examples/src/max_connection_age/server.rs

```diff
@@ -0,0 +1,42 @@
+use tonic::{transport::Server, Request, Response, Status};
```

Comment:
I don't think this is worth adding more examples.

Reply:
Removed that example code.